portal2/src/main/java/eu/europeana/portal2/web/presentation/model/PortalPageData.java

```diff
@@ -38,6 +38,8 @@
 
 	private boolean doTranslation = false;
 
+	private boolean isNofEnabled = false;
```

Comment:
According to code conventions this would be "nofEnabled", the getter/setter and all other code would stay the same...